### PR TITLE
Updated constants based on MNE-Python 0.19.1

### PR DESCRIPTION
**Pull request by ftadel**
Removed constants:
- FIFFB_MNE_EPOCHS_DROP_LOG
- FIFFB_MNE_EPOCHS_SELECTION
- FIFFB_MNE_RT_MEAS_INFO
- FIFFB_SSS_CAL_ADJUST

Renamed constants:
- FIFFB_SMSH_ASPECT => FIFFB_IAS_ASPECT
- FIFFB_SMSH_RAW_DATA => FIFFB_IAS_RAW_DATA

Added over 200 new constants defined in MNE-python, some of them needed to read linked FIF files.

### CODE REVIEW

**Comment by ftadel** [comment]
(I already pushed these modifications on the brainstorm repo, I need it for moving on with reading split FIF files...)

**Comment by agramfort** [comment]
Did you just copy the content of the python constants.py file ?

**Comment by ftadel** [comment]
Yes. I just added the ; and replaced # with %

Does it make sense?

File: matlab/fiff_define_constants.m [hunk 1]
```diff
@@ -1,118 +1,63 @@
...
-%    Revision 1.3  2006/04/10 23:26:53  msh
-%    Added fiff reading routines
-%
-%    Revision 1.2  2005/12/05 20:23:21  msh
-%    Added fiff_save_evoked. Improved error handling.
-%
-%    Revision 1.1  2005/12/05 16:01:04  msh
-%    Added an initial set of fiff writing routines.
-%
+function [FIFF, FWD] = fiff_define_constants()
```
Comment (agramfort) [inline matlab/fiff_define_constants.m]: Why did you change function signature ?

Comment (ftadel) [reply matlab/fiff_define_constants.m]: Because it had not been updated during the recent changes, meaning that it was already outdated. I replaced it with the short header from your python function.

We can restore this long history if you think it is useful. And maybe add it to your python function as well, since it was copied from there...

**Comment by agramfort** [comment]
thx @ftadel
